sw/device/silicon_creator/manuf/base/ft_personalize.c

```diff
@@ -164,6 +165,27 @@ static void log_self_hash(void) {
   // clang-format on
 }
 
+/*
+ * Return a pointer to the ROM_EXT manifest located in the slot b.
+ */
+static const manifest_t *rom_ext_manifest_b_get(void) {
+  return (const manifest_t *)(TOP_EARLGREY_EFLASH_BASE_ADDR +
+                              (TOP_EARLGREY_EFLASH_SIZE_BYTES / 2));
+}
+
+extern const uint32_t kCreatorSwCfgManufStateValue;
+
+/*
+ * Check if the `manuf_state_creator` field in the ROM_EXT manifest (slot b)
+ * matches the expected value that will be provisioned into the OTP.
+ */
+static status_t check_manuf_state_creator_binding(void) {
+  HARDENED_CHECK_EQ(
```

Comment:
This does not need to a be a hardened check. It can just be an ordinary `TRY_CHECK(x == y)` https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/lib/testing/test_framework/check.h;l=79?q=CHECK&ss=opentitan%2Fopentitan

Reply:
Ok, I've also replaced the hardened check used in the `check_otp_measurement_{pre,post}_lock` with the ordinary check. This should be more informative when the mismatch occurs.

Reply:
(LMK if you think a hardened check would be more suitable when checking against the locking status of OTP partition)